src/test/pybind/test_rados.py

```diff
@@ -1057,23 +1057,61 @@ def cb(_, buf):
     def test_lock(self):
         self.ioctx.lock_exclusive("foo", "lock", "locker", "desc_lock",
                                   10000, 0)
+        lockers_info = self.ioctx.list_lockers("foo", "lock")
+        eq(lockers_info["tag"], "")
+        eq(lockers_info["exclusive"], True)
+        eq(len(lockers_info["lockers"]), 1)
         assert_raises(ObjectExists,
                       self.ioctx.lock_exclusive,
                       "foo", "lock", "locker", "desc_lock", 10000, 0)
         self.ioctx.unlock("foo", "lock", "locker")
         assert_raises(ObjectNotFound, self.ioctx.unlock, "foo", "lock", "locker")
+        lockers_info = self.ioctx.list_lockers("foo", "lock")
+        eq(len(lockers_info["lockers"]), 0)
```

Comment:
I'd add asserts for `tag` and `exclusive` here for demonstration:
```suggestion
        eq(lockers_info["tag"], "")
        eq(lockers_info["exclusive"], True)
        eq(len(lockers_info["lockers"]), 0)
```